reinforcement_learning/Lesson-5-A2C-draft.ipynb

```diff
@@ -0,0 +1,1017 @@
+{
```

Comment:
<p>What about S_t+1 and R_t+1? What is being done?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BreakoutMentors/Data-Science-and-Machine-Learning/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>